cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateMaskedPaths(spec *rspec.Spec) error {
+	fmt.Println("validating maskedPaths")
+	for _, v := range spec.Linux.MaskedPaths {
+		fi, err := os.Stat(v)
+		if err != nil {
+			return err
+		}
+		if fi.Mode()&0444 != 0 {
+			return fmt.Errorf("%v should be readonly", v)
```

Comment:
@wking  spec says "mask over the provided paths inside the container so that they cannot be read."  it definitely says change paths permission to make them  can't be read.
So,  I think there is no need to "just try it".